cmd/run_zk_server.go

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+package main
+
+import (
+	"log"
+	"os"
+	"os/signal"
+	"syscall"
+
+	"github.com/facebookincubator/zk/integration"
+)
+
+func main() {
+
+	done := make(chan os.Signal, 1)
+	signal.Notify(done, os.Interrupt, syscall.SIGINT, syscall.SIGTERM)
+
+	cfg := integration.DefaultConfig()
+
+	server, err := integration.NewZKServer("3.6.2", cfg)
+	if err != nil {
+		log.Fatalf("unexpected error while initializing zk server: %v", err)
+	}
+
+	go func() {
+		if err = server.Run(); err != nil {
```

Comment:
@pmazzini we used to have an integration cycle like this, no?

Reply:
Yes, we have these tests in `conn_test.go`.

I don't really see why we need this inside a `cmd` directory.

Reply:
I'll remove this file.